presto-oracle/pom.xml

```diff
@@ -14,13 +14,20 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.oracle.version>19.3.0.0</dep.oracle.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>com.oracle.ojdbc</groupId>
             <artifactId>ojdbc8</artifactId>
-            <version>19.3.0.0</version>
+            <version>${dep.oracle.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId> com.oracle.ojdbc</groupId>
+            <artifactId>ucp</artifactId>
```

Comment:
This is being added in `Extract a variable with Oracle JDBC version` commit. Does not look intentional.

(Not sure why would you want to separate this btw, but maybe this was agreed upon)

Reply:
was asked here https://github.com/prestosql/presto/pull/3770#discussion_r429880053

Reply:
I think that at the time i suggested extracting the commit there was just one use of `dep.oracle.version`. But maybe I missed the other one.

Anyway. 
* Either leave commit which extract variable and only use it for `ojdbc8` entry. And then add `ucp` in followup commit together with pooling implementation.
* Or squash two commits together.

I am slightly toward former one.